### PR TITLE
Run styler

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

File: R/missing_argument_linter.R [hunk 1]
```diff
@@ -37,7 +37,8 @@ missing_argument_linter <- function(except = c("alist", "quote", "switch"), allo
     "self::EQ_SUB[following-sibling::*[not(self::COMMENT)][1][self::OP-RIGHT-PAREN or self::OP-COMMA]]"
   )
   if (!allow_trailing) {
-    conds <- c(conds,
+    conds <- c(
```
Comment (MichaelChirico) [inline R/missing_argument_linter.R:40]: is this a style guide violation? I like having both `conds` in the same line, I think it improves readability

File: R/settings.R [hunk 2]
```diff
@@ -159,7 +159,8 @@ validate_config_file <- function(config, config_file, defaults) {
     )
   }
 
-  validate_regex(config,
+  validate_regex(
```
Comment (MichaelChirico) [inline R/settings.R:162]: I'm more indifferent here but also didn't think this is a violation

File: R/missing_argument_linter.R (hunk 1 above)
Comment (IndrajeetPatil) [reply R/missing_argument_linter.R:40]: IMO, the point here is not whether the code violates the style guide or not, but rather that we don't fight against the tool (the formatter). 

If we start picking-and-choosing which changes to keep and which to discard, we lose the benefits of having a formatter: time saved and consistency achieved is offset by time lost in discussing some formatting changes and inconsistency introduced by personal preferences. 

Of course, we can always request changes upstream in the formatter if the team doesn't agree (e.g. we are already doing this for `switch()` statements).

Comment (MichaelChirico) [reply R/missing_argument_linter.R:40]: Don't really agree here TBH, this change actively makes the code less readable to me. "Don't fight the formatter" can apply where it's purely stylistic, e.g. the change in R/settings.R.

Here {styler} is actively violating what's allowed by the style guide, so yes it needs to be filed with them:

[list.files(path, pattern = "\\.[RrSsQq]$")](https://style.tidyverse.org/syntax.html#long-lines)

Comment (IndrajeetPatil) [reply R/missing_argument_linter.R]: Thanks, I have manually reverted these two changes.

**Review by MichaelChirico — APPROVED** [review]
thanks! hopefully there's an upstream update before the next time we run styler for a final decision

**Pull request merged**